includes/class-algolia-cli.php

```diff
@@ -131,9 +136,11 @@ private function do_reindex( Algolia_Index $index, $clear ) {
 
 		$progress = WP_CLI\Utils\make_progress_bar( sprintf( 'Processing %s pages of results.', $total_pages ), $total_pages );
 
-		$page = 1;
+		$page = $from_page;
 		do {
+			WP_CLI::log( sprintf('Indexing page %s.', $page );
 			$index->re_index( $page++ );
+			WP_CLI::log( sprintf('Indexed page %s.', ( $page - 1 ) );
```

Comment:
@moory-se I started testing your changes. And it looks like there are syntax errors on line 141 and 143. We need to add an extra closing bracket `)` before the semi-colon.